jekyll/_cci2/how-to-override-config.adoc

```diff
@@ -0,0 +1,344 @@
+
+[source,yaml]
+----
+# my-orb
+jobs:
+  custom-test:
+    docker:
+      - image: cimg/node:18.0
+    steps:
+      - checkout
+      - run: echo "Running custom test suite"
+      - run: npm test
+----
+
+Notice that this file:
+
+* Does NOT include `version: 2.1`.
+* Does NOT include `orbs:` or `workflows:` sections.
+* Contains only the `jobs:` section with the specific job definitions the team wants to override.
+
+=== 3. Add your URL orb prefix to your allow-list
+
+#ADD LINK TO ALLOW-LIST HOW-TO WHEN READY#
+
+=== 4. Override a job using the orb
+
+Add your URL orb to the `orbs` section of your configuration.
+
+Use the `override-with` key to replace the test job with a job from your orb. When the `main` workflow runs, CircleCI will use the custom-test job from your URL orb instead of your locally defined test job. If the `custom-test` job the orb does not exist, the `test` job configured in your main config will be used.
+
+[source,yaml]
+----
+version: 2.1
+
+orbs:
+  my-orb: https://raw.githubusercontent.com/my-org/my-repo/path/to/file/orb.yml
+
+jobs:
+  build:
+    docker:
+      - image: cimg/base:2023.03
+    steps:
+      - checkout
+      - run: echo "Building application"
+  test:
+    docker:
+      - image: cimg/base:2023.03
+    steps:
+      - checkout
+      - run: echo "Running standard tests"
+
+workflows:
+  main:
+    jobs:
+      - build
+      - test:
+        override-with: my-orb/custom-test
+        requires: build
+----
+
+[#real-world-template-example]
+== Real-world example: Platform team template with team overrides
+This example demonstrates how platform teams can maintain standardized pipelines, using config templates, while allowing development teams the flexibility to customize specific parts. The following list describes a centrally managed pipeline in which an application is built using a standard job and a test job is configured with a team-specific override:
+
+* When a pipeline triggers, the centralized template config is used.
+* The template imports the development team's custom job configuration as an orb.
+* The workflow runs the standard build job defined in the centralized template config.
+* For the test job, the configured override job (if it exists) is used instead of the template's standard test job.
+* The standard security-scan and deploy jobs continue to run as defined in the template config.
+
+=== 1. Create a centralized configuration template
+The platform team creates a central configuration (for example, `central-template.yml`) in a dedicated platform-team/ci-standards repository:
+
+[source,yaml]
+----
+version: 2.1
+
+# Import the team's config as an orb
+orbs:
+  team-config: << pipeline.parameters.config-override-url >>
+
+# Define parameters for configuration
+parameters:
+  config-override-url:
+    type: string
+    default: "https://raw.githubusercontent.com/default-team/project/main/.circleci/team-config.yml"
```

Comment:
`https://raw.githubusercontent.com/my-org/my-repo/refs/heads/main/.circleci/team-config.yml` is a more accurate example of a URL to fetch content from github. It has the `org` and `repo` components, and also needs the `refs/heads` part